renovate.json5

```diff
@@ -5,18 +5,20 @@
   ],
   "packageRules": [
     {
-      "paths": ["composer.json"],
-      "excludePackageNames": [
```

Comment:
configured, but p-r was opened. So try to use excludePackageNames instead of above.
https://github.com/line/line-bot-sdk-php/pull/399